cmd/gokey/main.go

```diff
@@ -30,6 +30,7 @@ func init() {
 	flag.StringVar(&realm, "r", "", "password/key realm (most probably purpose of the password/key)")
 	flag.StringVar(&output, "o", "", "output path to store generated key/password (default stdout)")
 	flag.BoolVar(&unsafe, "u", false, "UNSAFE: allow key generation without a seed")
+	flag.IntVar(&length, "l", 10, "characters of password or raw symmetric key bytes (default 10 chars for password and 32 bytes for raw")
```

Comment:
The description sounds a bit unclear. How about "number of characters in the generated password or number of bytes in the generated raw stream (default 10 for "pass" type and 32 for "raw" type)

Reply:
Resolved in 5562647